torch/testing/_internal/common_distributed.py

```diff
@@ -548,10 +548,10 @@ def wrapper(self):
     # Constructor patches current instance test method to
     # assume the role of the main process and join its subprocesses,
     # or run the underlying test function.
-    def __init__(self, method_name: str = "runTest") -> None:
-        super().__init__(method_name)
-        fn = getattr(self, method_name)
-        setattr(self, method_name, self.join_or_run(fn))
+    def __init__(self, methodName: str = "runTest") -> None:
```

Comment:
Is testslide requiring this `method_name` -> `methodName` change?

Reply:
`method_name` is wrong. PyThon's core unittest.TestCase uses `methodName`. The issue happens because testslide uses kwargs while other test frameworks args. So for other test frameworks, the incorrect name is okay